Streetcode/Streetcode.BLL/DTO/Sources/CreateSourceLinkCategoryDTO.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Streetcode.BLL.DTO.Sources
+{
+    public class CreateSourceLinkCategoryDTO
+    {
+        public int Id { get; set; }
```

Comment:
Having the same question as Vlad : why do we have sort of duplicate DTOs? And one more thing unclear: why some create DTOs have Id while some others do not?

Reply:
I deleted all unnecessary DTO